src/Temporalio/Converters/DefaultPayloadConverter.cs

```diff
@@ -56,7 +56,7 @@ public DefaultPayloadConverter()
         /// instances, so only subclasses would call this.
         /// </remarks>
         /// <seealso cref="DefaultPayloadConverter()" />
-        protected DefaultPayloadConverter(JsonSerializerOptions jsonSerializerOptions)
+        public DefaultPayloadConverter(JsonSerializerOptions jsonSerializerOptions)
```

Comment:
Can you make the one below this public too and remove the remarks paragraph from xmldoc on both that says why it is protected?